src/plugins/dashboard/public/dashboard_app/listing/dashboard_listing.tsx

```diff
@@ -282,6 +286,15 @@ export const DashboardListing = ({
           hasNoReference: referencesToExclude,
         })
         .then(({ total, hits }) => {
+          const searchEndTime = window.performance.now();
```

Comment:
I wonder if all this logic shouldn't live inside the `<TableListView />` component so other consumers would benefit from it.

We would have to pass the `analytics` plugin to the Kibana provider.

We could then have an additional (optional) prop (if not defined then no metric is sent).

```ts
performanceMetric={{
  // current metric
  dataFetchedLoadedTime: {
    enabled: true,
    meta: { ... }, // optional - any meta info for the metric
  },
  // other future metrics  
}}
```

WDYT?

Reply:
discussed this offline.

Reply:
As discussed offline let's keep it as implemented for now and we will revisit this later once we have better visibility of other use cases for metrics. 👍